TicketsDemo.CSV/Repositories/TrainCSVRepository.cs

```diff
@@ -0,0 +1,169 @@
+﻿using CsvHelper;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using TicketsDemo.Data.Entities;
+using TicketsDemo.Data.Repositories;
+
+namespace TicketsDemo.CSV.Repositories
+{
+    public class TrainCSVRepository : ITrainRepository
+    {
+        private string _trainPath = $@"{AppDomain.CurrentDomain.BaseDirectory}\App_Data\Trains.csv";
```

Comment:
please move file paths to some configuration service, such as 
ICSVConfiguration {
    string TrainsFilePath{get;set;}
    string CarriagesFilePath{get;set;}
    string PlacesFilePath{get;set;}
}